pipeline.Makefile

```diff
@@ -322,3 +328,38 @@ validate-clean:
 .PHONY: validate-debug
 validate-debug:
 	@:$(info $(DEBUG))
+
+# Mapping (Transformation) Goals
+# ==============================
+
+# Sentinel file to indicate mapping is complete
+MAPPING_SUCCESS_SENTINEL := $(MAPPING_OUTPUT_DIR)/_mapping_complete
+
+.PHONY: map-data
+map-data: $(MAPPING_SUCCESS_SENTINEL)
+
+$(MAPPING_SUCCESS_SENTINEL): $(SCHEMA_FILE) $(VALIDATION_SUCCESS_SENTINEL) $(DM_MAP_TARGET_SCHEMA) $(DM_TRANS_SPEC_DIR)
+	@echo "Running LinkML-Map transformation..."
+	@mkdir -p $(MAPPING_OUTPUT_DIR)
+	$(RUN) python ./src/dm_bip/map_data/map_data.py \
+		--source_schema $(SCHEMA_FILE) \
+		--target_schema $(DM_MAP_TARGET_SCHEMA) \
+		--data_dir $(DM_INPUT_DIR) \
+		--var_dir $(DM_TRANS_SPEC_DIR) \
+		--output_dir $(MAPPING_OUTPUT_DIR) \
+		--output_prefix $(DM_MAPPING_PREFIX) \
```

Comment:
What happens when some of these variables are empty? Will this command still work? i.e. does `--output_prefix` allow 0 or 1 arguments?

What needs to be defined, and what is okay to be empty?

Reply:
It should fail if any of those values are not defined, except for PREFIX and POSTFIX... But I should probably check and make sure.